deploy/foundation/hub/resources/webhook/webhook-validating-config.yaml

```diff
@@ -29,7 +29,6 @@ webhooks:
         apiVersions:
           - "v1"
         resources:
-          - clusterdeployments
```

Comment:
why this is removed?

Reply:
Currentlly, we will always sync clusterdeployments clusterset label by clusterpool or managedcluster's clusterset. 
So do not need to use webhook to check it anymore.